src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+import seedu.address.commons.exceptions.DataLoadingException;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.storage.JsonAddressBookStorage;
+
+/**
+ * Imports a data file from the specified path.
+ */
+public class ImportCommand extends Command {
+
+    public static final String COMMAND_WORD = "import";
+    public static final String MESSAGE_SUCCESS = "Data successfully imported";
+    public static final String MESSAGE_FILE_ERROR = "An error occurred during loading of data. The file "
+            + "might be corrupted.";
+    public static final String MESSAGE_FILE_MISSING = "The specified file does not exists";
```

Comment:
Should it be "exist" instead of "exists"?

Reply:
Fixed